src/main/kotlin/se/uulm/snowballr/backend/grpc/interceptor/AuthenticationInterceptor.kt

```diff
@@ -61,11 +65,14 @@ private val PUBLIC_METHODS =
  * This design supports browser-based gRPC clients using cookies for authentication and avoids an extra round-trip
  * by performing pre-emptive refresh. If the refresh token is also invalid or expired, the session is terminated.
  *
- * Note: Only methods listed in PUBLIC_METHODS or starting with the health check service name bypass authentication.
+ * Note: Only methods listed in [PUBLIC_METHODS] or [PUBLIC_SERVICES] bypass authentication.
  */
 @Suppress("ReturnCount")
```

Comment:
This isn't required anymore.
```suggestion
*/
```